otelcollector/deploy/dependentcharts/prometheus-node-exporter/README.md

```diff
@@ -41,6 +41,17 @@ helm upgrade [RELEASE_NAME] [CHART] --install
 
 _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documentation._
 
+### 3.x to 4.x
+
+Starting from version 4.0.0, the `node exporter` chart is using the [Kubernetes recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/). Therefore you have to delete the daemonset before you upgrade.
```

Comment:
Will this be an issue with 1P upgrading?

Reply:
yes it is. it already accounted for in my PR. This issue existed for both charts.

Reply:
and this comment is just from original repo. not changing it, as it will cause conflict every time.

Reply:
Ok sounds good thanks Vishwa

Reply:
I see your comment now that you removed the selector labels